pallets/transaction-multi-payment/src/lib.rs

```diff
@@ -36,65 +36,98 @@ use primitives::asset::AssetPair;
 use primitives::traits::{CurrencySwap, AMM};
 use primitives::{Amount, AssetId, Balance, CORE_ASSET_ID};
 
-use orml_utilities::with_transaction_result;
 use orml_utilities::OrderedSet;
 
 type NegativeImbalanceOf<C, T> = <C as Currency<<T as frame_system::Config>::AccountId>>::NegativeImbalance;
+// Re-export pallet items so that they can be accessed from the crate namespace.
+pub use pallet::*;
+
+#[frame_support::pallet]
+pub mod pallet {
+	use super::*;
+	use frame_support::pallet_prelude::*;
+	use frame_system::pallet_prelude::OriginFor;
+
+	#[pallet::pallet]
+	pub struct Pallet<T>(_);
+
+	#[pallet::hooks]
+	impl<T: Config> Hooks<T::BlockNumber> for Pallet<T> {
+		fn on_runtime_upgrade() -> frame_support::weights::Weight {
+			/*
+			mod previous {
```

Comment:
what happened here?

Reply:
one of the reasons why i just commented it out and did not bother with it - because i could not figure it out quickly how to rewrite this migration in the new way ( because it is not possible to mix the new pallet macro and the old decl_storage macro which is used here ).

But as it is already upgraded - it should be removed. See als nswer about the migration removal.

Reply:
Let's remove then?